src/main/java/dude/command/CommandExecutionException.java

```diff
@@ -0,0 +1,18 @@
+package dude.command;
```

Comment:
Could consider packaging exceptions into their own separate package from the command, for example they could be placed into `duke.exception`